src/v/coproc/router.cc

```diff
@@ -335,11 +336,11 @@ errc router::add_source(
     for (auto& [ntp, log] : logs) {
         auto found = _sources.find(ntp);
         if (found == _sources.end()) {
-            topic_state ts{
-              .log = log, .head = topic_offsets(), .scripts = {id}};
-            _sources.emplace(ntp, std::move(ts));
+            auto ts = ss::make_lw_shared<topic_state>(topic_state{
+              .log = log, .head = topic_offsets(), .scripts = {id}});
+            _sources.emplace(ntp, ts);
```

Comment:
this marked resolved, but nothing seems to have changed. did you intend to change it or respond with some info about it like maybe it was changed later?

Reply:
I hit resolve when I make the actual change, pending comments are like my TODO list, so when I hit resolve for me it means its been handled. I guess its confusing here because i was resolved in a future commit, I rebased into a later commit avoiding me from having to perform a conflict resolution.

Reply:
resolved in later commit

Reply:
i do the same thing, but the resolved state also signals to the reviewer that there was a state change on the comment. in this case, the state changed, but it is unclear what the state changed to.